typescript/cli/src/commands/options.ts

```diff
@@ -45,6 +45,7 @@ export const keyCommandOption: Options = {
 Dry-run: An address to simulate transaction signing on a forked network`,
   alias: 'k',
   default: ENV.HYP_KEY,
+  defaultDescription: '***REDACTED***',
```

Comment:
```suggestion
  defaultDescription: 'HYP_KEY env var',
```

Reply:
How about this to signal to user that's where the default will come from?

Reply:
Something similar sure– how about `process.env.HYP_KEY`?